### ISSUE TITLE
Improved Design For Footer

### ISSUE THREAD

**Issue by Asrani-Aman** [CircuitVerse/CircuitVerse#4563]
### Is your feature request related to a problem? Please describe

Revamping the Footer Section
1.Arranging the uneven Margin 
2.adding hover effect on the links for the better user experience

### Describe the solution you'd like

Revamping the Footer Section

Attaching the Figma Link

https://www.figma.com/file/656lhewR01Ax4C7QiNAPZY/Untitled?type=design&node-id=0%3A1&mode=design&t=NFQ6vXJBjKEuXnm9-1

BEFORE DEKTOP VIEW FOOTER DESIGN
<img width="1680" alt="Screenshot 2024-01-09 at 10 09 52 AM" src="https://github.com/CircuitVerse/CircuitVerse/assets/96644946/62daecb1-8c7a-4635-9d28-1e0b00dc096e">

After Desktop View Footer Design

https://github.com/CircuitVerse/CircuitVerse/assets/96644946/a7e5e2a0-b230-48b8-99a3-96b6f5813f40

Before Mobile View 
<img width="389" alt="Screenshot 2024-01-10 at 10 49 53 PM" src="https://github.com/CircuitVerse/CircuitVerse/assets/96644946/0f8d7fa1-7ab8-4512-8974-cfc59a6025c3">

After Mobile View 

https://github.com/CircuitVerse/CircuitVerse/assets/96644946/d91d08a9-c22b-470e-9fd6-2c46842b4057

### Are you working on this?

Yes

**Comment by Asrani-Aman** [comment]
@tanmoysrt sir @Prerna-0202 ma'am @tachyons sir can i work on this issue?

**Comment by nitin10s** [comment]
Nice spot, we removed a couple of logos from the footer which left it looking uneven. Yes, the desktop new design makes sense. Also, the mobile view is correct in the current one itself, lets not change it.

**Comment by nitin10s** [comment]
Also, check these discrepancies: https://circuitverse-team.slack.com/archives/CQ17325A5/p1705214789678159

**Comment by Asrani-Aman** [comment]
@nitin10s sir so should i start working for the desktop design? if yes, can you please assign this issue to me

**Comment by upsaurav12** [comment]
@nitin10s @Asrani-Aman i think can put logps(not sponsor logo) to the right of the of thr circuitVerse logo for mobile view because there is empty space in the right of the circuitverse logo which is not looking good

**Comment by Asrani-Aman** [comment]
@tanmoysrt bhaiya @nitin10s bhaiya , for the desktop view should I make a PR as you said above , 
and i have just add a one more design for the footer in the same[ Figma Link](https://www.figma.com/file/656lhewR01Ax4C7QiNAPZY/Untitled?type=design&node-id=0%3A1&mode=design&t=PQfCJFIIJ42YeHdX-1)

bhaiya, can you please suggest which one is better ?

**Comment by tanmoysrt** [comment]
> @tanmoysrt bhaiya @nitin10s bhaiya , for the desktop view should I make a PR as you said above , and i have just add a one more design for the footer in the same[ Figma Link](https://www.figma.com/file/656lhewR01Ax4C7QiNAPZY/Untitled?type=design&node-id=0%3A1&mode=design&t=PQfCJFIIJ42YeHdX-1)
> 
> bhaiya, can you please suggest which one is better ?

I have missed this issue.
You can raise PR

**Comment by tanmoysrt** [comment]
1st design looks simple.

**Comment by Asrani-Aman** [comment]
@tanmoysrt bhaiya, so i should work on first design or second design?

**Comment by tanmoysrt** [comment]
> @tanmoysrt bhaiya, so i should work on first design or second design?

Design of Frame 10 & 20 looks good, you can go with that

**Comment by Asrani-Aman** [comment]
@Bhavishya2601 you can work on this issue and can create a new PR

Make sure you don't use particular values of margin and padding 
also there is some PR's for this, which we can not merge as they were too old, you can refer them

I am assigning it to you

**Comment by Bhavishya2601** [comment]
@Asrani-Aman pls check out the pr

**Comment by AnushkaMaheshwari4** [comment]
hello! i am currently learning frontend and would love to work on this issue under gsoc2025. it seems beginner friendly and i'd love to give it a try. kindly assign if availavble.

**Comment by divyam-jha123** [comment]
@Asrani-Aman  ,I would love to work on this issue if its still available.

**Comment by Asrani-Aman** [comment]
can you propose the figma first?

**Comment by divyam-jha123** [comment]
https://www.figma.com/design/656lhewR01Ax4C7QiNAPZY/Untitled?node-id=0-1&t=CqbsowrVbDizC7Lw-1
After first design here looks good. It spread the elements evenly in the space there.
What do you think?? @Asrani-Aman  @nitin10s

**Comment by shirshamay** [comment]
Hello, I saw this issue and would like to help i saw its open, however i see comments about people who asked to helped. If its still open I would like to solve this. I am a first time contributor with strong CSS,HTML JS and TS. Can I please assigned?

**Comment by Mithilkamane** [comment]
Hi @Asrani-Aman , can I work on improving the footer design and fixing alignment + spacing issues?
I can make it responsive using flexbox and update the styling cleanly. 
Please assign this to me if it's still available. Thanks!

**Comment by Teesta-Mukherjee** [comment]
Hi @Asrani-Aman 👋  
I’d like to work on this issue and implement the footer design as per the provided Figma file.  
Please let me know if I can proceed. Thanks!

**Comment by tachyons** [comment]
If you want to work on this or any other UI design issue, start with a design prototype.

**Comment by Mithilkamane** [comment]
Hi @tachyons, thanks for the clarification!
I’ll start by creating a design prototype for the footer based on the provided Figma and share it here for review.
Once it’s approved, I’ll proceed with improving alignment, spacing, and making the footer fully responsive using flexbox.
Please let me know if that works. Thanks

**Comment by Mithilkamane** [comment]
<img width="1024" height="1536" alt="Image" src="https://github.com/user-attachments/assets/90d125f8-3d3d-4829-80dc-a97a047eeef2" /> 
Hi @tachyons 
I’ve created a simple footer design prototype (desktop & mobile) to show the intended layout, spacing, and responsiveness.
Please let me know if this looks good so I can proceed with the HTML/CSS implementation.  
Thanks!

**Comment by iamundefinedguy** [comment]
@upsaurav12 yeah design is good in my pov, but try to make the links center in mobile version it would be better UX

**Comment by Mithilkamane** [comment]
<img width="1024" height="1536" alt="Image" src="https://github.com/user-attachments/assets/168fc8be-ec56-4e7c-a192-c12040b15873" /> 
@upsaurav12 @tachyons 
I’ve updated the mobile version to center-align the links for better UX and shared the revised prototype above.
 let me know if this looks good or if any other tweaks are needed.

**Comment by RoshniSingh0116** [comment]
Hi! I’d like to work on this issue.
I’ll implement the footer updates based on the provided Figma design.
Please assign it to me.

**Comment by Irfan762** [comment]
@Asrani-Aman  could you please assign this issue to me?
